src/GovUk.Education.ExploreEducationStatistics.Data.Model/GovUk.Education.ExploreEducationStatistics.Data.Model.csproj

```diff
@@ -12,10 +12,7 @@
 
   <ItemGroup>
     <PackageReference Include="GeoJSON.Net" Version="1.4.1" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.SqlServer" Version="8.0.7" />
-    <PackageReference Include="Newtonsoft.Json" Version="13.0.3" />
-    <PackageReference Include="Thinktecture.EntityFrameworkCore.BulkOperations" Version="8.1.1" />
-  </ItemGroup>
+    </ItemGroup>
```

Comment:
Minor one but the formatting doesn't look right here. Thought I'd point it out given that there's been an effort to sort out the consistency of formatting elsewhere!